apigee-test-shared/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "apigee-shared-test",
+  "version": "0.0.1",
+  "description": "Used to test apigee proxies",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "chai": "4.2.0",
+    "mocha": "5.2.0",
+    "nyc": "13.2.0",
+    "sinon": "7.2.3"
+  },
+  "scripts": {
+    "test": "nyc --all --reporter=html --reporter=text mocha --recursive tests"
```

Comment:
👍 thanks for updating

Reply:
I don't know this `nyc` thing so I'm taking y'all on your word that this is working as expected. :)

Reply:
`nyc` is the test coverage tool for Javascript